tsconfig.build.json

```diff
@@ -0,0 +1,19 @@
+{
+  "compilerOptions": {
+    "declaration": true,
+    "module": "commonjs",
+    "esModuleInterop": true,
+    "allowSyntheticDefaultImports": true,
+    "moduleResolution": "node",
+    "noImplicitAny": true,
+    "strict": true,
+    "preserveSymlinks": true,
+    "allowJs": true,
+    "noLib": false,
```

Comment:
this is the default for `noLib`, I think? https://www.typescriptlang.org/docs/handbook/compiler-options.html